pifi/driver.py

```diff
@@ -0,0 +1,85 @@
+class APA102Driver():
+    # LED Settings
+    __MOSI_PIN = 10
+    __SCLK_PIN = 11
+    __LED_ORDER = 'rbg'
+
+    def __init__(self, led_settings, clear_screen=False):
+        from apa102_pi.driver import apa102
```

Comment:
>we can do dynamic imports in Python, right? we can do the imports conditionally depending on the driver, and separate the drivers into 2 files

I like this if it's possible! Furthermore, we should modify the install_dependencies.sh script to install the things that you need. Perhaps ideally we could detect the config settings from their config.json and install the appropriate dependencies, but I don't view this as necessary to merge (this might require writing a python script that instantiates an LedSettings instance from the config and prints out the configured driver so that bash can read it).